packages/connect-form/src/components/connect-form-actions.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  spacing,
+  uiColors,
+} from '@mongodb-js/compass-components';
+
+const formActionStyles = css({
+  padding: spacing[4],
+  borderTop: `1px solid ${uiColors.gray.light2}`,
+  textAlign: 'right',
+});
+
+function ConnectFormActions({
+  onConnectClicked,
+}: {
+  onConnectClicked: () => void;
+}): React.ReactElement {
+  return (
+    <div className={formActionStyles}>
+      <Button variant={ButtonVariant.Primary} onClick={onConnectClicked}>
+        Connect
+      </Button>
```

Comment:
Should this button be disabled / hidden when the URL is not valid? Currently it is clickable and when pressed you are being connected to the last valid url that was in the input

Reply:
Yes, but I think it probably makes sense to do that in a separate pr if that's cool with you. We can add some tests too.